### PR TITLE
Clarify interop with pre-1.3 cipher suite registry

### PR DESCRIPTION
**Pull request by richsalz**

### CODE REVIEW

**Review by kaduk — CHANGES_REQUESTED** [review]
The registered ciphersuite values have always been distinct two-octet values, with no internal structure or interpretation to the separate octets.

A similar statement to this might be relevant for the SignatureScheme registry, which combines the preexisting SignatureAlgorithm and HashAlgorithm codepoint spaces into a new registry without internal structure going forward but constrained by historical legacy.

**Comment by richsalz** [comment]
I was focused on interop, I don't see where/how to put that sentence you want.

**Comment by richsalz** [comment]
Did I get the 1.2/1.3 bytes backwards, hence Sean's confusion?

File: draft-ietf-tls-iana-registry-updates.md [hunk 1]
```diff
@@ -315,6 +315,11 @@ TLS_ECDHE_PSK_WITH_CHACHA20_POLY1305_SHA256   | {0xCC,0xAC}
 TLS_DHE_PSK_WITH_CHACHA20_POLY1305_SHA256     | {0xCC,0xAD}
 ~~~
 
+To emphasize, this registry contains _both_ TLS 1.3 and pre-1.3 cipher
+suites. Therefore, registraries for TLS 1.2 and earlier may require
```
Comment (jsalowey) [inline draft-ietf-tls-iana-registry-updates.md:319]: Should this be **registrations?**

Comment (jsalowey) [reply draft-ietf-tls-iana-registry-updates.md:319]: I don't understand where the requirement to reserve and entire byte range in TLS 1.2 comes from?  Can you elaborate?

**Comment by jsalowey** [comment]
I think this pull request should be closed as it is not correct.

**Pull request closed without merging**